apps/molgenis-components/src/components/tables/DataDisplayCell.vue

```diff
@@ -52,10 +52,10 @@ export default defineComponent({
     cellTypeComponentName() {
       return this.isArrayType
         ? "ListDisplay"
-        : typeMap[this.metaData.columnType] || "StringDisplay";
+        : typeMap[this.metaData?.columnType] || "StringDisplay";
```

Comment:
metadata is required, so I do not approve of this monkey patch. This should be fixed at the input side.